### PR TITLE
Add new test for resource_name

### PR DESCRIPTION
**Pull request by gergocs**
JerryScript-DCO-1.0-Signed-off-by: Gergo Csizi csgergo92@gmail.com

### CODE REVIEW

**Comment by gergocs** [comment]
Patch is ready for review

File: jerry-core/lit/lit-magic-strings.inc.h [hunk 1]
```diff
@@ -781,7 +781,10 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_SET_UTC_DATE_UL, "setUTCDate")
 #if JERRY_BUILTIN_STRING && JERRY_ESNEXT
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_STARTS_WITH, "startsWith")
 #endif
+#if JERRY_RESOURCE_NAME \
+|| !(JERRY_RESOURCE_NAME)
```
Comment (zherczeg) [inline jerry-core/lit/lit-magic-strings.inc.h:785]: I know this is auto-generated but `if (something || !something)` is totally meaningless.

File: tests/unit-core/test-resource-name.c [hunk 2]
```diff
@@ -166,13 +179,36 @@ main (void)
...
+  program = jerry_parse ((const jerry_char_t *) source_5,
+                         strlen (source_5),
+                         &parse_options);
+  TEST_ASSERT (!jerry_value_is_error (program));
+
+  resource_value = jerry_get_resource_name (program);
+  compare_result = jerry_binary_operation (JERRY_BIN_OP_STRICT_EQUAL, resource_value, parse_options.resource_name);
+  TEST_ASSERT (jerry_value_is_true (compare_result));
+
+  jerry_release_value (resource_value);
```
Comment (zherczeg) [inline tests/unit-core/test-resource-name.c:206]: There is no newline after the assert in the other cases.

**Review by zherczeg — APPROVED** [review]
LGTM with a minor change

**Comment by gergocs** [comment]
Patch is ready for review

**Review by rerobika — APPROVED** [review]

**Pull request merged**